db/migrate/20150114024350_create_course_announcements.rb

```diff
@@ -0,0 +1,14 @@
+class CreateCourseAnnouncements < ActiveRecord::Migration
+  def change
+    create_table :course_announcements do |t|
+      t.references :course, null: false
+      t.string :title, null: false
+      t.text :content
```

Comment:
Comparing between `announcement.content` vs `announcement.description` tells me that either works. If there must be a choice, I agree with @wangqiang1208  that content is more accurate, while the description might be used to describe the Announcement